torch/_subclasses/fake_tensor.py

```diff
@@ -1349,7 +1360,8 @@ def _cached_dispatch_impl(
                 if self.cache_crosscheck_enabled:
                     # For debugging / testing: Validate that the output synthesized
                     # from the cache matches the output created by normal dispatch.
-                    self._crosscheck_cache_output(output, func, types, args, kwargs)
+                    with disable_fake_tensor_cache(self):
```

Comment:
What's going on here?

Reply:
This code is trying to get the correct value. If I dont turn off fake tensor, for HOPS, it uses the fake tensor cache for subgraph run. Earlier, this was not a problem because we should just run an op here.

Reply:
I see what's going on. Thanks for the explanation